src/Data/Loader/UserLoader.php

```diff
@@ -74,7 +74,7 @@ public function get_public_users( array $keys ) {
 
 		global $wpdb;
 
-		$results = $wpdb->get_results(
+		$results = $wpdb->get_results( // phpcs:ignore WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.DirectDatabaseQuery.DirectQuery
```

Comment:
@jasonbahl should this be cached?

Reply:
@justlevine seems reasonable to make this cached.

Reply:
Created issue so we don't lose this: https://github.com/wp-graphql/wp-graphql/issues/2476